kuma/api/v1/tests/test_plus.py

```diff
@@ -93,3 +93,22 @@ def test_ping_landing_page_survey_authenticated(user_client, wiki_user):
     assert response.status_code == 200
     (result,) = LandingPageSurvey.objects.all()
     assert result.user == wiki_user
+
+
+@pytest.mark.django_db
+def test_landing_page_variant_happy_path(client, settings):
+    settings.PLUS_PRICE_VARIANTS = ["Foo", "Bar", "Buzz"]
```

Comment:
Nit. I would suggest doing the following, just so we also test that we've configured the casting/conversion of the env var properly. 
```suggestion
    settings.PLUS_PRICE_VARIANTS_JSON = '["Foo", "Bar", "Buzz"]'
```

Reply:
I'm actually more in favor of using `kuma/settings/pytest.py` then the test suite can never be at the mercy of individual people's overrides of that setting.